backend/src/BackendOnlyStdLib/LibDarkInternal.fs

```diff
@@ -798,4 +798,29 @@ human-readable data."
           | _ -> incorrectArgs ())
       sqlSpec = NotQueryable
       previewable = Impure
+      deprecated = NotDeprecated }
+
+
+
+    { name = fn "DarkInternal" "parseAndExecuteExpr" 0
+      typeParams = []
+      parameters =
+        [ Param.make "code" TString ""; Param.make "userInputs" (TDict TString) "" ]
+      returnType =
+        // TODO: improve type
+        TResult(TVariable "a", TString)
```

Comment:
TODO: declaration doesn't match reality. also, maybe there's a better structure to return than this

Reply:
decided to just always return a string